### PR TITLE
:sparkles: Fixes upon review comments

### PR DESCRIPTION
**Pull request by andreyod**
## Related issue(s)

Fixes #

### CODE REVIEW

File: cmd/mailbox-controller/controller.go [hunk 1]
```diff
@@ -356,26 +372,30 @@ func (ctl *mbCtl) ensureBinding(ctx context.Context, workspace *tenancyv1alpha1.
...
 		logger.Error(err, "object does not appear to be a provider's copy of a consumer's object", "syncTarget", st.Name)
-		return ""
+		return "", err
 	}
 	spaceID := ctl.kbSpaceRelation.SpaceIDFromKubeBind(kbSpaceID)
 	if spaceID == "" {
-		logger.Error(nil, "failed to get consumer space ID from a provider's copy", "syncTarget", st.Name)
-		return ""
+		logger.Error(errNoSpaceId, "failed to get consumer space ID from a provider's copy", "syncTarget", st.Name)
+		return "", errNoSpaceId
```
[the remark this answers is not in the document]
Comment (andreyod) [reply cmd/mailbox-controller/controller.go:385]: Line 70. New error.

File: cmd/mailbox-controller/controller.go [hunk 2]
```diff
@@ -144,13 +146,18 @@ func (ctl *mbCtl) enqueue(obj any) {
 		logger.V(4).Info("Enqueuing reference due to workspace", "wsName", typed.Name)
 		ctl.queue.Add(typed.Name)
 	case *edgev2alpha1.SyncTarget:
-		mbwsName := ctl.mbwsNameOfSynctarget(typed)
-		if mbwsName == "" {
-			logger.Error(nil, "Failed to construct mailbox workspace name from SyncTarget", "syncTargetName", typed.Name)
-			return
+		mbwsName, err := ctl.mbwsNameOfSynctarget(typed)
+		if err != nil {
```
Comment (ezrasilvera) [inline cmd/mailbox-controller/controller.go:150]: maybe changing the condition order will be more clear ? 
If (err) {
  If != errNoSpaceId {
    return
    }
    ctl.queue.Add(obj)
 } 
 else {
 }

Comment (andreyod) [reply cmd/mailbox-controller/controller.go:150]: I didn't want to return from switch, maybe someone will add some code after the switch-case.

**Pull request merged**